io.github.yuxshao.ptcollab.yml

```
@@ -0,0 +1,21 @@
+id: io.github.yuxshao.ptcollab
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: ptcollab
+rename-icon: ptcollab
+rename-desktop-file: ptcollab.desktop
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

since you removed wayland.

---

done